ladon.go

```diff
@@ -49,6 +57,9 @@ func (l *Ladon) auditLogger() AuditLogger {
 func (l *Ladon) IsAllowed(r *Request) (err error) {
 	policies, err := l.Manager.FindRequestCandidates(r)
 	if err != nil {
+
+		// We return no match
+		go l.Metric.RequestNoMatch(*r)
```

Comment:
Hm, wouldn't it make more sense to also have a function for errors? This isn't really no match, but something out of the ordinary